tests/test_new_version_diff.py

```diff
@@ -0,0 +1,90 @@
+import pytest
```

Comment:
Please incorporate these new tests in `test_new_version.py` following the style with `unittest`

Reply:
I removed that line... the test works with pytest, is there anything else I need to do?